src/pages/whats-new.js

```diff
@@ -0,0 +1,131 @@
+import React from "react"
+import { Container, Row, Col, ListGroup } from 'react-bootstrap';
+import { Link, Element } from "react-scroll";
+import { BLOCKS } from "@contentful/rich-text-types"
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer'
+
+// import twitterIcon from '../images/icons/twitter-icon-blue.svg'
+import NewFeature from "../components/new-feature"
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+import "../styles/pages/whats-new.scss"
+
+// https://dev.to/flexdinesh/accessing-nested-objects-in-javascript--9m4
+//  Access Nested Objects Using Array Reduce 
+const getNestedObject = (nestedObj, pathArr) => {
+    return pathArr.reduce((obj, key) =>
+        (obj && obj[key] !== 'undefined') ? obj[key] : undefined, nestedObj);
+}
+
+//  formatting date from Contenful, 2020-06-03 => June 3, 2020
+const dateToString = (date, locale) => {
+    const monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+    const monthNamesES = ['Enero', 'Febrero', 'Marzo', 'Abril', 'Mayo', 'Junio', 'Julio', 'Agosto', 'Septiembre', 'Octubre', 'Noviembre', 'Diciembre']
+    var mydate = new Date(date); 
+
+    return ( 
+            locale ==='en' ? 
+            monthNames[mydate.getMonth()] +" " + (mydate.getDate()+1) : 
+            (mydate.getDate()+1) + " de " + monthNamesES[mydate.getMonth()]
+            ) +", "+ mydate.getFullYear()
+}
+
+
+// formatting the date (June 3, 2020) into an id that react scroll can reference (#june-3-2020)
+const dateToDivID = (date) => {return "#" + date.replace(/[\W_]+/g, '-').toLowerCase()}
+
+// options to replace formatting for rich text
+const options = {
+    renderNode: {
+        [BLOCKS.PARAGRAPH]: (node, children) =>  children,
```

Comment:
what do the options do?

Reply:
when rendering rich text with `documentToReactComponents` it wraps it in a `<p>` so this removes it!

Reply:
oh yeah I think we can delete the JSON file too

Reply:
I see. thanks for explaining!